Makefile.host

```diff
@@ -0,0 +1,39 @@
+#
+# Common Makefile for host
+#
+# Copyright 2018-2021 Phoenix Systems
+#
+# %LICENSE%
+#
+
+CROSS ?=
+
+MKDEP = $(CROSS)gcc -MM
+MKDEPFLAGS = $(CFLAGS)
+
+CC = $(CROSS)gcc
+
+ifeq ($(DEBUG), 1)
+	CFLAGS += -Og
+else
+	CFLAGS += -O2 -DNDEBUG
+endif
+
+CFLAGS += -Wall -Wstrict-prototypes -g -fomit-frame-pointer
+
+AR = $(CROSS)ar
+ARFLAGS = -r
+
+LD = $(CROSS)gcc
+LDFLAGS += -Wl,--gc-sections
+LDLIBS += -pthread
```

Comment:
Uch, we probably don't need pthreads for every binary either, but it's not that important - we can fix it in the consecutive commits with other buildsystem changes.